tests/Unit/API/WP/NotificationsServiceTest.php

```diff
@@ -5,6 +5,7 @@
 
 use Automattic\WooCommerce\Admin\RemoteInboxNotifications\TransformerService;
 use Automattic\WooCommerce\GoogleListingsAndAds\API\WP\NotificationsService;
+use Automattic\WooCommerce\GoogleListingsAndAds\HelperTraits\SyncTrait;
```

Comment:
This is no longer used as the functions are included with `NotificationsService`

Reply:
https://github.com/woocommerce/google-listings-and-ads/pull/2804/commits/b98f3aa3375554b6f780ec50352e53583f2f4e7e